lib/matplotlib/widgets.py

```diff
@@ -1934,8 +1934,15 @@ def onmove(self, event):
             self.linev.set_visible(False)
             self.lineh.set_visible(False)
             if self.needclear:
+                if self.useblit and self.background is not None:
+                    self.canvas.restore_region(self.background)
+                else:
+                    self.canvas.draw()
+                    self.canvas.restore_region(self.background)
+                    self.canvas.blit(self.ax.bbox)
```

Comment:
Since `useblit` may be `False` here, I think it does not make sense to use `blit`.